pkg/kv/kvserver/kvflowcontrol/replica_rac2/processor.go

```diff
@@ -313,7 +261,7 @@ type Processor interface {
 	// processing for MsgStorageAppend.
 	//
 	// raftMu is held.
-	HandleRaftReadyRaftMuLocked(context.Context, rac2.RaftEvent)
+	HandleRaftReadyRaftMuLocked(context.Context, RaftNodeBasicState, rac2.RaftEvent)
```

Comment:
Isn't all the info in `RaftBasicState` already incorporated into `RaftEvent`? Or should it be?